contracts/Core.sol

```diff
@@ -91,13 +91,17 @@ contract Core is CoreInterface, Util {
 	 * @param _chainIdOrigin origin chain id
 	 * @param _chainIdRemote remote chain id
 	 * @param _openSTRemote remote openSTUtility/openSTValue contract address
+	 * @param _blockHeight block height
```

Comment:
Those type of documentations are not useful at all ... It's just separates variable name into two part ...

Reply:
Updated with detailed comments.